src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -55,8 +55,8 @@
   --#{$empty-state}--m-xs__footer--MarginBlockStart: var(--pf-t--global--spacer--md);
 
   // Actions
-  --#{$empty-state}__actions--RowGap: var(--pf-t--global--spacer--sm);
-  --#{$empty-state}__actions--ColumnGap: var(--pf-t--global--spacer--md);
+  --#{$empty-state}__actions--RowGap: var(--pf-t--global--spacer--gap--group--vertical);
+  --#{$empty-state}__actions--ColumnGap: var(--pf-t--global--spacer--gap--action-to-action--default);
 }
 
```

Comment:
I'm wondering if we should we update the empty state footer to use the gap-to-gap spacer? It's the spacer between action groups in the footer https://github.com/patternfly/patternfly/blob/7bd4dd2a1eb955b621ac3bafc46265b4d782ade8/src/patternfly/components/EmptyState/empty-state.scss#L53

What it looks like now with a small spacer. Doesn't really differentiate between the groups
<img width="1371" alt="Screenshot 2024-07-01 at 4 22 15 PM" src="https://github.com/patternfly/patternfly/assets/35148959/cf9ff030-13d6-4402-84e4-4994a6ab9cda">

And with the group-to-group spacer
<img width="1473" alt="Screenshot 2024-07-01 at 4 23 18 PM" src="https://github.com/patternfly/patternfly/assets/35148959/9dc99c8d-7d16-49f1-84cb-8d51293bdbb6">

If not, assuming the goal of the small spacer is to keep a tight group like the primary and other actions are part of the same group, should we also update it to use `--pf-t--global--spacer--gap--group--vertical` as you've done here?

Reply:
I like this update. Within the link button group, it should get gap--group--vertical, but if the primary action is in a different group, it should get group-to-group.